examples/snake/main.go

```diff
@@ -0,0 +1,193 @@
+package main
+
+import (
+	"fmt"
+	"image/color"
+	"log"
+	"math/rand"
+
+	"github.com/hajimehoshi/ebiten"
+	"github.com/hajimehoshi/ebiten/ebitenutil"
+	"github.com/hajimehoshi/ebiten/inpututil"
+)
+
+const (
+	screenWidth  = 640
+	screenHeight = 480
+	xNumInScreen = screenWidth / gridSize
+	yNumInScreen = screenHeight / gridSize
+	gridSize     = 10
+)
+
+const (
+	dirLeft = iota + 1
+	dirRight
+	dirDown
+	dirUp
+)
+
+type Position struct {
+	X int64
+	Y int64
+}
+
+type Game struct {
+	moveDirection int8
+	appleX        int64
+	appleY        int64
+	snakeBody     []Position
+	timer         uint64
+	speed         int8
+	score         int64
+	bestScore     int64
+	level         int8
```

Comment:
All the integers can be int. Specifying a bit size is not needed.

Reply:
done